keyboards/skyloong/gk104/q1/q1.c

```diff
@@ -0,0 +1,297 @@
+//  Copyright 2023 NaturalZh (@NaturalZh)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "quantum.h"
+
+#define HC595_ST_PIN B1  //74HC595 storage register clock input
+#define HC595_SH_PIN B0  //74HC595 shift register clock input
+#define HC595_DS B3   // 74HC595 serial data input
+#include "led_hc595.c"
+uint8_t IND = 0;  //buffer of LED Display
```

Comment:
c files should not be included, and should be added by `SRC += ` in rules.mk (or post_rules.mk).